LoadingUI/LoadingUI/LoadingViewController.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  LoadingViewController.swift
+//  LoadingUI
+//
+//  Created by Cora Jacobson on 10/5/20.
+//
+
+import UIKit
+import SceneKit
+
+public class LoadingViewController: UIViewController {
+    
+    private var scnView: SCNView!
+    private var scnScene: SCNScene!
+    private var cameraNode: SCNNode!
+    
+    private var drawingController = DrawingController()
+    
+    public var duration: Double = 5
```

Comment:
I personally prefer putting public properties at the top of the class above my private properties.